persistence/src/tests/spi/clusterstatetest.cpp

```diff
@@ -169,7 +172,7 @@ ClusterStateTest::testReady()
 {
     lib::ClusterState s("version:1 storage:3 distributor:3");
 
-    Bucket b(document::Bucket(document::BucketSpace::placeHolder(), document::BucketId(16, 1)), PartitionId(0));
+    Bucket b(makeBucket(document::BucketId(16, 1)));
```

Comment:
To align with `makeDocumentBucket`'s naming convention maybe we should rename this to `makeSpiBucket`? Though I doubt it will cause ambiguities since it's just used in the SPI module.